wkcuber/cubing.py

```diff
@@ -175,7 +229,16 @@ def cubing_job(args):
 
 def cubing(source_path, target_path, layer_name, dtype, batch_size, args=None) -> dict:
 
-    target_wkw_info = WkwDatasetInfo(target_path, layer_name, dtype, 1)
+    target_mag = Mag(getattr(args, "target_mag", 1))
```

Comment:
Why not `target_mag = Mag(args.target_mag)` and then defaulting to target_mag to 1 in the argparse definition?